tests/testsuite/profile_config.rs

```diff
@@ -173,6 +173,38 @@ found package specs: bar, bar:0.5.0",
         .run();
 }
 
+#[cargo_test]
+fn profile_config_rustflags() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+            cargo-features = ["profile-rustflags"]
+
+            [profile.dev]
+            rustflags = ["-C", "link-dead-code=yes"]
+
+            [package]
+            name = "foo"
+            version = "0.0.1"
+            "#,
+        )
+        .file("src/main.rs", "fn main() {}")
+        .build();
+
+    p.cargo("build -v")
+        .masquerade_as_nightly_cargo()
+        .env_remove("CARGO_INCREMENTAL")
```

Comment:
Was this line necessary to get this to pass? If not then this should be removed, if so then this should be built in to other parts of cargo's test harness. IIRC there's already a function that does a bit of sanitization for the environment of child proceses.

Reply:
Assuming you were referring to `env_remove("CARGO_INCREMENTAL")`, it was not needed.